sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Tests/AzureSdkLoggingTests.cs

```diff
@@ -11,201 +11,225 @@
 using System.Threading.Tasks;
 using Azure.Core.Diagnostics;
 using Azure.Core.TestFramework;
-using Microsoft.AspNetCore.Builder;
+using Azure.Monitor.OpenTelemetry.Exporter.Models;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
 using Xunit;
-using static Xunit.CustomXunitAttributes;
 
 namespace Azure.Monitor.OpenTelemetry.AspNetCore.Tests
 {
     public class AzureSdkLoggingTests
     {
-        [Theory(Skip = "Test is unstable and need to be re-written without the need to create a web server.")]
+        private readonly MockTransport _mockTransport = new MockTransport(_ => new MockResponse(200).SetContent("ok"));
+
+        [Theory]
         [InlineData(false, LogLevel.Debug, null)]
-        [InlineData(false, LogLevel.Information, null)]
+        //[InlineData(false, LogLevel.Information, null)] // TODO: This test needs to be fixed. It's failing because the CustomEvent changes.
```

Comment:
This one test is failing. I think because of the CustomEvent changes. I'll touch base with Vishwesh tomorrow.

Reply:
Include an additional check here on "Azure.Monitor.OpenTelemetry.CustomEvents" (rule.CategoryName != "Azure.Monitor.OpenTelemetry.CustomEvents") - https://github.com/Azure/azure-sdk-for-net/blob/1c5804ebd4479cf6ebd480355d1075290fe71488/sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Internals/AzureSdkCompat/AzureEventSourceLogForwarder.cs#L33

Reply:
Thanks @rajkumar-rangaraj , I fixed this in our internal copy of the AzureEventSourceLogForwarder. 
Are we going to have issues with the public copy?

Reply:
No changes are needed in public copy. This internal copy is customized to track logs with warning level and above.

Reply:
> internal copy is customized to track logs with warning level and above.

This may not be working as intended. CustomEvents is configured for Information, which is below Warning.
https://github.com/Azure/azure-sdk-for-net/blob/669ea53427f57f49661f6747462eaaeecdfcbb63/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/CustomEvents/ApplicationInsightsEventLogger.cs#L40

Reply:
This gets called only with Azure Event Source events and not with ILogger logs.